internal/querycoordv2/balance/rowcount_based_balancer.go

```diff
@@ -96,6 +96,12 @@ func (b *RowCountBasedBalancer) Balance() ([]SegmentAssignPlan, []ChannelAssignP
 		replicas := b.meta.ReplicaManager.GetByCollection(cid)
 		for _, replica := range replicas {
 			splans, cplans := b.balanceReplica(replica)
+			if len(splans) > 0 || len(cplans) > 0 {
+				log.Info("nodes info in replica",
+					zap.Int64("collection", replica.CollectionID),
+					zap.Int64("replica", replica.ID),
+					zap.Int64s("nodes", replica.GetNodes()))
```

Comment:
prefer to adjust the log level to debug, or this will print many logs with many nodes